cpp/SnowflakeS3Client.cpp

```diff
@@ -76,8 +76,8 @@ SnowflakeS3Client::SnowflakeS3Client(StageInfo *stageInfo, unsigned int parallel
     Aws::String(stageInfo->credentials.at(AWS_SECRET_KEY)),
     Aws::String(stageInfo->credentials.at(AWS_TOKEN)));
 
-  s3Client = new Aws::S3::S3Client(credentials, clientConfiguration);
-
+  s3Client = new Aws::S3::S3Client(credentials, clientConfiguration,
+          Aws::Client::AWSAuthV4Signer::PayloadSigningPolicy::Never, true);
```

Comment:
can you please add a comment for `true`? By looking at this code, it is not clear what this means until we check the reference doc.

Reply:
That's a good point!